skimage/morphology/tests/test_binary.py

```diff
@@ -8,21 +8,24 @@
 
 
 img = color.rgb2gray(data.astronaut())
-bw_img = img > 100
+bw_img = img > 100./255
 
+def assert_internally_equal(a, b, border=1):
```

Comment:
please use ``_assert_internally_equal`` since this is a helper function that won't be exposed elsewhere.